### ISSUE TITLE
BoxedDestroyed warning under Node.js 12

### ISSUE THREAD

**Issue by majorz** [romgrk/node-gtk#127]
I am getting a number of BoxedDestroyed warnings under Node.js 12. I tried it on different 12.x versions and different CPU architectures reproducing it each time.

A minimal snippet I isolated this to is:
```
gi = require('node-gtk')
gobject = gi.require('GObject', '2.0')
global.gc()
```

And when running `node --expose_gc issue.js` I get the warnings:
```
[WARN] boxed.cc: BoxedDestroyed: 235: boxed possibly not freed (GIRepository.Typelib : void)
[WARN] boxed.cc: BoxedDestroyed: 235: boxed possibly not freed (GIRepository.Typelib : void)
```

**Comment by romgrk** [comment]
Yes, those warning are not a leak, `GIRepository.Typelib` objects are not handled by `node-gtk` AFAICT. I'd prefer to leave the warning in place until memory management is deemed complete enough, should be after #87 is fixed. Closing as duplicate of that one.
